### DISCUSSION TITLE
Local network Gateway creation

### DISCUSSION THREAD

**Discussion by benoitmonnier** [Azure/Enterprise-Scale#1358] [Q&A]
Hello,

Is it possible to create local network gateways and connections?

Thanks in advance for your help

Regards

**Comment by jtracey93** [comment]
Hey,

Yes it is possible by following this https://learn.microsoft.com/azure/vpn-gateway/tutorial-site-to-site-portal#LocalNetworkGateway

Thanks

**Comment by benoitmonnier** [comment]
Yes, but I'm talking about creating it with the terraform caf entreprise scale.

Has this been planned?

**Reply by jtracey93** [reply]
This isn't something we have planned today, but please raise an issue on that repo as a feature request for us to consider, and proivder some context/justification :)

https://aka.ms/alz/tf/repo
